src/txn/txn_rollback_to_stable.c

```diff
@@ -248,17 +253,23 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
         /* Stop processing when we find a stable update according to the given timestamp. */
         if (durable_ts <= rollback_timestamp) {
             __wt_verbose(session, WT_VERB_RTS,
-              "%s :History store update valid with durable timestamp: %s and stable timestamp: %s",
-              S2BT(session)->dhandle->name, __wt_timestamp_to_string(durable_ts, ts_string[0]),
-              __wt_timestamp_to_string(rollback_timestamp, ts_string[1]));
+              "%s :History store update valid with start timestamp: %s, durable timestamp: %s, "
+              "stop timestamp: %s and stable timestamp: %s",
+              S2BT(session)->dhandle->name, __wt_timestamp_to_string(hs_start_ts, ts_string[0]),
+              __wt_timestamp_to_string(durable_ts, ts_string[1]),
+              __wt_timestamp_to_string(hs_stop_ts, ts_string[2]),
+              __wt_timestamp_to_string(rollback_timestamp, ts_string[3]));
             valid_update_found = true;
             break;
         }
 
         __wt_verbose(session, WT_VERB_RTS,
-          "%s :History store update aborted with durable timestamp: %s and stable timestamp: %s",
-          S2BT(session)->dhandle->name, __wt_timestamp_to_string(durable_ts, ts_string[0]),
-          __wt_timestamp_to_string(rollback_timestamp, ts_string[1]));
+          "%s :History store update aborted with start timestamp: %s, durable timestamp: %s, stop "
+          "timestamp: %s and stable timestamp: %s",
+          S2BT(session)->dhandle->name, __wt_timestamp_to_string(hs_start_ts, ts_string[0]),
+          __wt_timestamp_to_string(durable_ts, ts_string[1]),
+          __wt_timestamp_to_string(hs_stop_ts, ts_string[2]),
+          __wt_timestamp_to_string(rollback_timestamp, ts_string[3]));
```

Comment:
You don't need the object name in the messages.

Reply:
Thanks, I corrected all the verbose messages as per this comment.